package common.protocol.L2G;

message AnnounceLinkInfo
{
	required int32 link_id = 1;
}
